docs/wxm-devices/wifi-m5-bundle/m5-install-weather-station.mdx

```diff
@@ -115,6 +116,11 @@ The pole together with the outdoor sensor should be placed in an installation po
 
 <hr/>
 
+:::note
+All of 1, 2 and 3 are important tips for measuring temperature and humidity as accurate as possible
+while 3. is important for avoiding faulty precipitation and wind measurements.
+:::
+
```

Comment:
Please move this note above the `<hr/>` tag above (the horizontal line in the doc).